File: LENTABP10/src/main/java/com/lenta/lentabp10/models/ExciseStampRestInfo.java
package com.lenta.lentabp10.models;

/// <summary>
/// Акцизная марка, элемент списка IT_MARKS в SaveTaskDataToSapRestRequest
/// </summary>

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

public class ExciseStampRestInfo {
    /// Номер набора (null для ненабора)
    @JsonProperty("MATNR_OSN")
    @Getter private String matnrOsn;

    /// Номер товара
    @JsonProperty("MATNR")
    @Getter private String matnr;

    /// Причина движения
    @JsonProperty("GRUND")
    @Getter private String writeOffCause;

    /// Код акцизной марки
    @JsonProperty("PDF417")
    @Getter private String stamp;

    /// Признак bad mark
    @JsonProperty("REG")
    @Getter private String reg;

    public ExciseStampRestInfo(String matnrOsn, String matnr, String writeOffCause, String stamp, String reg) {
        this.matnrOsn = matnrOsn;
        this.matnr = matnr;
        this.writeOffCause = writeOffCause;
        this.stamp = stamp;
        this.reg = reg;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/models/ExciseStampRestInfo.kt
package com.lenta.bp10.models

import com.google.gson.annotations.SerializedName

class ExciseStampRestInfo(
        /// Номер набора (null для ненабора)
        @SerializedName("MATNR_OSN")
        val matnrOsn: String,

        /// Номер товара
        @SerializedName("MATNR")
        val matnr: String,

        /// Причина движения
        @SerializedName("GRUND")
        val writeOffCause: String,

        /// Код акцизной марки
        @SerializedName("PDF417")
        val stamp: String,

        /// Признак bad mark
        @SerializedName("REG")
        val reg: String
)